apisix/include/apisix/model/pubsub.proto

```diff
@@ -55,16 +73,18 @@ message CmdEmpty {}
 message PubSubReq {
     int64 sequence = 1;
     oneof req {
-        CmdEmpty cmd_empty = 31;
-        CmdPing cmd_ping = 32;
+        CmdEmpty cmd_empty                       = 31;
```

Comment:
We can remove cmd_empty  which is test-only? Using `cmd_kafka_fetch` in pubsub.t is enough.

Reply:
@spacewander This would make the `pubsub` module test the relevant code that relies on `kafka`, and I'm not sure if I should do that.

Reply:
What about adding a comment to show that this cmd is test-only?

Reply:
After rechecking, I found that CmdEmpty has added a test-only flag.

https://github.com/apache/apisix/blob/d95500948d0af71bb6a378b9634d57b984c6e78c/apisix/include/apisix/model/pubsub.proto#L35-L39